src/LayerEditor/ui/data/abstract_model.py

```diff
@@ -0,0 +1,59 @@
+from LayerEditor.ui.tools import undo
+from LayerEditor.ui.tools.id_map import IdMap, IdObject
+
+
+class AbstractModel:
```

Comment:
Radeji bych to asi  prejmenoval na AbstractListModel.

Reply:
List by mohl být matoucí. Třída používá dict a existuje metoda která seřadí prvky a vrátí list. Mohu přejmenovat na AbstractDictModel.